src/client.h

```diff
@@ -345,6 +345,9 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	{ return m_proto_ver; }
 
 	bool connectedToServer();
+	void confirmRegister();
+	bool m_is_registration_confirmation_state = false;
```

Comment:
don't make those attributes public, make them private

Reply:
But it is used in `Game::connectToServer()`
```cpp
if (client->m_is_registration_confirmation_state) {
	// ...
} else {
	// ...
}
```